src/arcam/fmj/state.py

```diff
@@ -39,13 +40,13 @@ class State():
     _state: Dict[int, Optional[bytes]]
     _presets: Dict[int, PresetDetail]
 
-    def __init__(self, client: Client, zn: int):
+    def __init__(self, client: Client, zn: int, api_model: ApiModel = ApiModel.API450_SERIES):
```

Comment:
Added this extra param here for testing purposes. If needed, i can mock `_update_amxduet` instead to specify the api model.

Reply:
OK